h/schemas/admin_group.py

```diff
@@ -17,18 +17,39 @@
 _ = i18n.TranslationString
 
 VALID_GROUP_TYPES = (
-    ('private', _('Private')),
     ('restricted', _('Restricted')),
     ('open', _('Open')),
 )
 
 
+def _split_origins(value):
+    if value != colander.null:
+        return [origin.strip() for origin in value.splitlines()]
```

Comment:
This can produce empty values if the input contains empty lines. I suggest those should just be stripped out with `[... if origin.strip()]`.